runelite-client/src/main/java/net/runelite/client/plugins/keyremapping/KeyRemappingListener.java

```diff
@@ -68,7 +68,7 @@ public void keyTyped(KeyEvent e)
 	@Override
 	public void keyPressed(KeyEvent e)
 	{
-		if (client.getGameState() == GameState.LOGIN_SCREEN || !plugin.chatboxFocused())
+		if (client.getGameState() == GameState.LOGIN_SCREEN || client.getGameState() == GameState.LOGIN_SCREEN_AUTHENTICATOR || !plugin.chatboxFocused())
```

Comment:
This change should be made for all plugins, not just the keyremapping plugin. Move this change to KeyManager.

Reply:
I don't think the key remapper uses the key manager though, since it has its own remap logic?